lib/get-all-tweets.js

```diff
@@ -1,15 +1,61 @@
 const Twitter = require('twitter');
 
-module.exports = function () {
-	let client = new Twitter({
+module.exports = function getAllTweets() {
+	// function writeTweetsToFile(tweets, filepath = 'tmp/all-tweets.json') {
```

Comment:
commented out this function because it look like the `writeTweetsToFile` func is not used at all

Reply:
yup, just added it when i was testing offline before using dynamodb